pkg/build/builder/docker.go

```diff
@@ -116,11 +116,11 @@ func (d *DockerBuilder) Build() error {
 		if authPresent {
 			glog.V(4).Infof("Authenticating Docker push with user %q", pushAuthConfig.Username)
 		}
-		glog.V(1).Infof("Pushing image %s ...", pushTag)
+		glog.V(0).Infof("Pushing image %s ...", pushTag)
```

Comment:
not your fault but this message format and the message format in sti.go are different, can you sync them?